package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@dfinity/ic-js",
-  "version": "0.17.1",
+  "version": "0.17.2",
```

Comment:
You can revert this, v0.17.1 is already the next version.

Reply:
Done, thanks